### DISCUSSION TITLE
Categorization Add On

### DISCUSSION THREAD

**Discussion by aimanskie** [exoplatform/exo-community#67] [Q&A]
Hi community, i want to know how to add this add on https://www.exoplatform.com/blog/categorize-your-spaces-easily-with-this-new-exo-platform-add-on/ as we are interested in categorizing the spaces. I am using community version 6.4 and so far i don't see it in the application registry.

Appreciate any insight on this. Thanks!

**Comment by rdenarie** [comment]
Hello,
This addon is a very old one, for old eXo version, it is no longer relevant
